src/components/victory-scatter/victory-scatter.jsx

```diff
@@ -327,16 +383,58 @@ export default class VictoryScatter extends React.Component {
     // a new `VictoryScatter` with nearly identical props, except (1) tweened
     // and (2) `animate` set to null so we don't recurse forever.
     if (this.props.animate) {
-      // Do less work by having `VictoryAnimation` tween only values that
-      // make sense to tween. In the future, allow customization of animated
-      // prop whitelist/blacklist?
-      const animateData = pick(this.props, [
-        "data", "domain", "height", "maxBubbleSize", "padding", "samples", "size",
-        "style", "width", "x", "y"
-      ]);
+      let animateData;
+
+      // If nodes need to exit, all we want to do is to fade them out.
+      if (this.props.exitingNodes) {
+        animateData = {
+          data: this.props.data.map((datum, idx) => {
+            const key = (datum.key || idx).toString();
+            return this.props.exitingNodes[key] ?
+              Object.assign({}, datum, { opacity: 0 }) :
```

Comment:
We should extract the logic for the actual transition. For pie it will be the slice angle going to zero, for bar the height going to zero etc. It would be nice if this could be something that the user could define too, but that can be supported later.